frontend/apps/crates/entry/jig/edit/src/edit/publish/dom/mod.rs

```diff
@@ -240,7 +240,12 @@ fn render_page(state: Rc<Publish>) -> Dom {
                     .child(html!("button-rect", {
                         .text(STR_PUBLISH)
                         .text(state.jig.focus_display())
-                        .property("disabled", state.jig.modules.lock_ref().iter().find(|m| !m.is_complete).is_some())
+                        .property(
```

Comment:
Even though the teacher won't be able to navigate to the publish screen via the sidebar, if they know to add /publish to the url, they'll see the screen. This is a safety check.